API does not work on Ubuntu servers

On Ubuntu servers, I try to start the Explorer API code and get this result:

```
jrush@unchainedindex:~/D/t/api|master✓➤ yarn start
yarn run v1.22.4
$ nodemon server.js
[nodemon] 1.19.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching dir(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node server.js`
internal/modules/cjs/loader.js:1033
  throw err;
  ^

Error: Cannot find module './webSockets'
Require stack:
- /home/jrush/Development/trueblocks-explorer/api/websockets/index.js
- /home/jrush/Development/trueblocks-explorer/api/chifra_stream_events.js
- /home/jrush/Development/trueblocks-explorer/api/server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1030:15)
    at Function.Module._load (internal/modules/cjs/loader.js:899:27)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/home/jrush/Development/trueblocks-explorer/api/websockets/index.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/jrush/Development/trueblocks-explorer/api/websockets/index.js',
    '/home/jrush/Development/trueblocks-explorer/api/chifra_stream_events.js',
    '/home/jrush/Development/trueblocks-explorer/api/server.js'
  ]
}
[nodemon] app crashed - waiting for file changes before starting...
^C⏎
```

I'm not sure, but it may be the case that the code is case sensitive on Ubuntu? I tried changing `webSockets` to `websockets` (lower case) and it changed the error message (still didn't work).

--- a/api/websockets/index.js
+++ b/api/websockets/index.js
@@ -1,5 +1,5 @@
 const progressModule = require('./progress');
-const webSockets = require('./webSockets');
+const webSockets = require('./websockets');
 
 module.exports = {
   reportProgress: progressModule.createReportProgress(webSockets),

